sci-libs/rocBLAS/rocBLAS-5.7.1-r1.ebuild

```diff
@@ -0,0 +1,109 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DOCS_BUILDER="doxygen"
+DOCS_DIR="docs"
+DOCS_DEPEND="media-gfx/graphviz"
+ROCM_VERSION=${PV}
+inherit cmake docs edo multiprocessing rocm
+
+DESCRIPTION="AMD's library for BLAS on ROCm"
+HOMEPAGE="https://github.com/ROCmSoftwarePlatform/rocBLAS"
+SRC_URI="https://github.com/ROCmSoftwarePlatform/rocBLAS/archive/rocm-${PV}.tar.gz -> rocm-${P}.tar.gz"
+S="${WORKDIR}/${PN}-rocm-${PV}"
+
+LICENSE="BSD"
+KEYWORDS="~amd64"
+SLOT="0/$(ver_cut 1-2)"
+IUSE="benchmark test"
+REQUIRED_USE="${ROCM_REQUIRED_USE}"
+
+RESTRICT="test" # Tests fail
```

Comment:
I just observe only a trivial Fortran header include error mentioned at  https://github.com/gentoo/gentoo/pull/33400#issuecomment-1773730274. After copying headers from `clients/include/rocblas/internal` to `clients/include` at `${BUILD_DIR}`, everything if fine. I guess adding a include flag (probably an upstream bug) is the fix.

I passed all enabled tests on a single MI210 machine with `sci-libs/blis-0.9.0` as reference. OpenBLAS and netlib reference segfault somehow, probably they have bug.